src/main/java/com/browserstack/automate/ci/jenkins/AutomateTestAction.java

```diff
@@ -112,6 +113,9 @@ private Session getSession(BrowserStackCredentials credentials, ProjectType proj
         } else {
             client = new AutomateClient(credentials.getUsername(), credentials.getDecryptedAccesskey());
         }
+        if(JenkinsProxySettings.hasProxy()){
```

Comment:
try the `optimize imports and rearrange code` in other files as well...

Reply:
Done